### DISCUSSION TITLE
Windows OS Support

### DISCUSSION THREAD

**Discussion by graphixillusion** [cupcakearmy/autorestic#45] [Q&A] [answered]
Are you planning to release a Windows's build too?

**Comment by cupcakearmy** [comment, marked as answer]
I would love to, however there are few limitations at least rn.
The biggest issue is that I can't really test/develop it.
Another thing would be the path stuff that works differently in windows, like exec paths, delimiters and so on.

**Reply by cupcakearmy** [reply]
windows support is being worked on, could come in the next weeks https://github.com/cupcakearmy/autorestic/pull/68
